### ISSUE TITLE
get csv query stuck during teardown

### ISSUE THREAD

**Issue by vasukulkarni** [red-hat-storage/ocs-ci#2942]
seen in Dev CI runs  https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/job/ocs-ci/51/execution/node/72/log/

the query should timeout sooner if at all needed in teardown (we probably dont even need to run this query)

```
08:33:35 - MainThread - ocs_ci.ocs.ocp - WARNING - Failed to get resource: ocs-operator.v4.6.0-85.ci of kind: csv, selector: None, Error: Error during execution of command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n openshift-storage -o yaml.
Error is Error from server (NotFound): clusterserviceversions.operators.coreos.com "ocs-operator.v4.6.0-85.ci" not found

08:33:35 - MainThread - ocs_ci.ocs.ocp - WARNING - Number of attempts to get resource reached!
08:33:35 - MainThread - ocs_ci.ocs.ocp - INFO - Cannot find resource object ocs-operator.v4.6.0-85.ci
08:33:35 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 5 seconds before next iteration
08:33:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n openshift-storage -o yaml
08:33:40 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Error from server (NotFound): clusterserviceversions.operators.coreos.com "ocs-operator.v4.6.0-85.ci" not found

08:33:40 - MainThread - ocs_ci.ocs.ocp - WARNING - Failed to get resource: ocs-operator.v4.6.0-85.ci of kind: csv, selector: None, Error: Error during execution of command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n openshift-storage -o yaml.
Error is Error from server (NotFound): clusterserviceversions.operators.coreos.com "ocs-operator.v4.6.0-85.ci" not found

08:33:40 - MainThread - ocs_ci.ocs.ocp - WARNING - Number of attempts to get resource reached!
08:33:40 - MainThread - ocs_ci.ocs.ocp - INFO - Cannot find resource object ocs-operator.v4.6.0-85.ci
08:33:40 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 5 seconds before next iteration
08:33:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n openshift-storage -o yaml
08:33:46 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Error from server (NotFound): clusterserviceversions.operators.coreos.com "ocs-operator.v4.6.0-85.ci" not found

08:33:46 - MainThread - ocs_ci.ocs.ocp - WARNING - Failed to get resource: ocs-operator.v4.6.0-85.ci of kind: csv, selector: None, Error: Error during execution of command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n openshift-storage -o yaml.
Error is Error from server (NotFound): clusterserviceversions.operators.coreos.com "ocs-operator.v4.6.0-85.ci" not found

08:33:46 - MainThread - ocs_ci.ocs.ocp - WARNING - Number of attempts to get resource reached!
08:33:46 - MainThread - ocs_ci.ocs.ocp - INFO - Cannot find resource object ocs-operator.v4.6.0-85.ci
08:33:46 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 5 seconds before next iteration
08:33:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig cluster/auth/kubeconfig get csv ocs-operator.v4.6.0-85.ci -n 
```

**Comment by vavuthu** [comment]
https://github.com/red-hat-storage/ocs-ci/pull/2898 will address this issue as well.

**Comment by petr-balogh** [comment]
As we merged the #2898 I think we can close this one
